src/Entities/Message.php

```diff
@@ -38,9 +38,11 @@
  * @link https://core.telegram.org/bots/api#message
  *
  * @method int                                    getMessageId()                              Unique message identifier
- * @method int                                    getMessageThreadId()                        Optional. Unique identifier of a message thread to which the message belongs; for supergroups only
+ * @method int                                    getMessageThreadId()                        Optional. Unique identifier of a message thread to which the message belongs;
+for supergroups only
```

Comment:
That newline might be a mistake...?

Reply:
Ah yes, the forced newline when auto-formatting, ned to find a way to disable that. We should probably look into pint anyway for formatting.